client/src/components/Profile/Profile.jsx

```diff
@@ -1,11 +1,23 @@
 import React from 'react';
-import styled from 'styled-components';
+import styled ,{keyframes}from 'styled-components';
 import Review from '../Review/Review.jsx';
 import AverageReview from '../AverageReview/AverageReview.jsx';
 import ReviewBar from '../ReviewBar/ReviewBar.jsx';
 
+
+const open =keyframes`
+  from{
+    transform : translateY(40%);
+  }
+
+  to{
+    transform:translateY(0);
+  }
+`;
```

Comment:
Interesting using key frames. Gonna look into this.

Reply:
yeah the problem i have is that i cant add animation when i close the modal, im trying to solve this my conditionally giving different prob value but what actually happens with modal is that it gets created and destroyed when you open and close the modal so i cant pass in a state/prop value to styled components.